homeassistant/components/switchbot/__init__.py

```diff
@@ -1 +1,97 @@
-"""The switchbot component."""
+"""Support for Switchbot devices."""
+from asyncio import Lock
+
+import switchbot  # pylint: disable=import-error
```

Comment:
```suggestion
import switchbot
```

This shouldn't be needed if we have the requirements set correctly in `manifest.json`

Reply:
The library is "force commented" from the hassio requirements file. According to @frenck this is done due to issues with hassio commit tests workflow.